Bottom: 86b8ac5c10d1bc7c6b3eb278789b34be10bc5db0
Top:    cb7414a9acc1b3bcfcfd3d513feb68c56c6782bc
Author: Ben Greear <greearb@candelatech.com>
Date:   2014-11-04 15:26:58 -0800

mac80211: print mac addr when authenticating.

Helps compare with logging info in drivers when using
multiple vifs.

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/net/mac80211/mlme.c b/net/mac80211/mlme.c
index 9861550..bae5f6e 100644
--- a/net/mac80211/mlme.c
+++ b/net/mac80211/mlme.c
@@ -4564,8 +4564,9 @@ int ieee80211_mgd_auth(struct ieee80211_sub_if_data *sdata,
 	{
 		struct timeval tv;
 		do_gettimeofday(&tv);
-		sdata_info(sdata, "authenticate with %pM at: %lu.%lu\n",
-			   req->bss->bssid, tv.tv_sec, tv.tv_usec);
+		sdata_info(sdata, "%pM authenticate with %pM at: %lu.%lu\n",
+			   sdata->dev->dev_addr, req->bss->bssid,
+			   tv.tv_sec, tv.tv_usec);
 	}
 
 	err = ieee80211_prep_connection(sdata, req->bss, false, false);
